platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -733,7 +763,7 @@ aziotedge_check() {
     eval container_size='$'"$(echo "$ARCH"_iotedge_container_size)"
     TOTAL_SIZE=$(echo $binary_size $container_size | awk '{print $1 + $2}')
     if [ -z "$MOUNTPOINT" ]; then
-        wrap_debug "No Storage Path Provided, Using Present working directory"
+        wrap_debug_message "No Storage Path Provided, Using Present working directory"
```

Comment:
Instead of  `Present working directory`, could you just print out the `$(pwd)` please?
(Or is that too much hassle given the nature of the cross Oses.)

Reply:
Good point!